.idea/index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello world!</h1>
```

Comment:
Actually a base struncure of htmp page should looks like this https://www.w3schools.com/html/html_intro.asp (paragrahp: A Simple HTML Document)

Reply:
Also please add link to github merge request to make me see an repository